package.json

```diff
@@ -23,16 +23,18 @@
   "contributors": [
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
+  "types": "types/index.d.ts",
   "files": [
     "lib/",
     "index.js",
     "html.js"
   ],
   "dependencies": {
-    "fault": "^1.0.0"
+    "fault": "^1.0.0",
+    "micromark": "~2.11.0"
```

Comment:
I think that micromark should stay as a devDependency since it's only used in test files

Reply:
We’ve had this sort of discussion before between other unified members. The conclusion was that if the dependency is used for its types only, it belongs in `dependencies`, not `devDependencies`.

I must admin that in this particular case I doubted this was needed, because micromark extensions are basically always used with an explicit dependency on the `micromark` package. However, I noticed the `micromark-extension-gfm-*` extensions already had this dependency, so I might as well add it here as well.

Reply:
> The conclusion was that if the dependency is used for its types only, it belongs in dependencies, not devDependencies.

I’m not 100% agreeing here. It can work for certain things. But like: I don’t want to add `unified` to every plugin. Plugins can’t work without unified, so someone always adds them to one.
And I think for micromark is the same

Reply:
> always used with an explicit dependency on the micromark package.

That’s true but I’m not 100% a fan of it. I’d rather move towards extensions that don’t include micromark (instead using “third-party” utilities)